ppoprf/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+The code in this repository is still being actively developed, and is thus regarded as suitable only for experimental purposes.
+
+## Reporting a Vulnerability
+
+Please report any observed issues to the [GitHub repository](https://github.com/brave-experiments/sta-rs/). 
```

Comment:
Please remove the trailing whitespace at the end of this line.

Git flags this, although github does not. Probably all the copies need the same fix.